Orange/widgets/data/owdatasets.py

```diff
@@ -125,6 +123,12 @@ def __init__(self):
         super().__init__()
         self.local_cache_path = os.path.join(data_dir(), self.DATASET_DIR)
 
+        self._header_labels = [header['label'] for _, header in self.HEADER_SCHEMA]
+        self._html_description_tags = ['year', 'source', 'title', 'description', 'seealso', 'references']
+
+        self._header_index = namedtuple('_header_index', [*[info_tag for info_tag, _ in self.HEADER_SCHEMA]])
```

Comment:
This is a syntax error on older python versions (as reported by failing tests).
But also, I don't see a reason for unpacking and repacking the list...

Reply:
Thanks, forgot to remove this when I was testing something.